### DISCUSSION TITLE
Pressing "enter" when focused on a directory puts me in a weird place

### DISCUSSION THREAD

**Discussion by yorunoken** [sxyazi/yazi#1245] [1. Q&A] [answered]
just installed Yazi, pressing "enter" 
![image](https://github.com/sxyazi/yazi/assets/69751661/084622fd-5196-4679-90e8-8b5cfc2dc210)
on the left can be seen what happens when I press enter on a folder. On the right is how it's normally done.
I can leave by pressing ctrl+c a few times then typing "quit"

**Comment by sxyazi** [comment, marked as answer]
Use `:q!` to quit Vim, see https://stackoverflow.com/questions/11828270/how-do-i-exit-vim

**Reply by yorunoken** [reply]
oh, it's vim commands? That's nice, but why does this screen exist? 🤔
